pkg/labels/labels.go

```diff
@@ -370,10 +381,17 @@ func parseSource(str string) (src, next string) {
 	sourceSplit := strings.SplitN(str, ":", 2)
 	if len(sourceSplit) != 2 {
 		next = sourceSplit[0]
-		if strings.HasPrefix(next, common.ReservedLabelKey) {
-			src = common.ReservedLabelSource
-			next = strings.TrimPrefix(next, common.ReservedLabelKey)
+		if strings.HasPrefix(next, common.BaseLabelSourceExtPrefix) {
```

Comment:
@tgraf So we can translate back from the extended version of the label source.
For example: `cilium.src.k8s.foo=bar` -> `k8s:foo=bar`

if we have only `k8s.foo=bar` I can't really be sure which is the source or not.